src/components/totals/FundsRaised/__tests__/FundsRaised-test.js

```diff
@@ -239,4 +239,20 @@ describe('FundsRaised', function() {
     });
   });
 
+  describe('takes groupValues property', function() {
```

Comment:
Seems like a good idea to reword it a bit. This file in general has a bit of an assortment with regard to how `describe`s have been written in particular. In particular, lines 210 and 226.

Could reword them all now in this PR?